lib/pavilion/plugins/system/host_name.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+import pavilion.varaibles
+import pavilion.system_plugins as system_plugins
+
+class HostName( system_plugins.SystemPlugins ):
+
+    def __init__( self, sub_keys=[ None ] ):
```

Comment:
The `subkeys = [None]` is bad form. I'll comment more on this below at the __init__ definition.